parser/src/cfg/defcfg.rs

```diff
@@ -39,6 +39,8 @@ pub struct CfgOptions {
         target_os = "unknown"
     ))]
     pub windows_interception_mouse_hwid: Option<[u8; HWID_ARR_SZ]>,
+    #[cfg(target_os = "macos")]
```

Comment:
```suggestion
    #[cfg(any(target_os = "macos", target_os = "unknown"))]
```

This should be also compile on wasm, so it works in "external tools" aka just my vscode-kanata extension so far.